app/translations.js

```diff
@@ -295,6 +295,16 @@ const translations = {
     'hsl_travel_card': 'HSL card',
     'is-open': 'Geöffnet:',
     'itinerary-co2.title': 'CO₂-Emissionen der Reise',
+    'itinerary-co2.title-sr': 'Kohlenstoffdioxid-Emissionen der Reise',
+    'itinerary-co2.description-simple':
+      '{co2value} g CO₂ Emissionen werden durch diese Reise verursacht.',
+    'itinerary-co2.description-simple-sr':
+      '{co2value} g Kohlenstoffdioxid Emissionen werden durch diese Reise verursacht.',
+    'itinerary-co2.description-sr':
+      '{co2value} g Kohlenstoffdioxid Emissionen werden durch diese Reise verursacht. Ein Auto würde {carCo2Value} g Kohlenstoffdioxid auf derselben Reise ausstoßen.',
+    'itinerary-co2.description':
+      '{co2value} g CO₂ Emissionen werden durch diese Reise verursacht. Ein Auto würde {carCo2Value} g CO₂ auf derselben Reise ausstoßen.',
+    'itinerary-co2.link': 'So reduzieren wir Emissionen ›',
```

Comment:
Our explanation page focuses on the explanation of the calculus, so I'd suggest to adapt the wording a bit:
```suggestion
    'itinerary-co2.link': 'So berechnen wir die Emissionen ›',
```